scripts/g.extension/g.extension.py

```diff
@@ -234,6 +233,51 @@ def urlopen(url, *args, **kwargs):
     return urlrequest.urlopen(request, *args, **kwargs)
 
 
+def get_github_branches(
+    github_api_url="https://api.github.com/repos/OSGeo/grass-addons/branches",
+    version_only=True,
+):
+    """Get ordered list of branch names in repo using github API
+    For the official repo we assume that at least one version branch is present"""
+    req = urlrequest.urlopen(github_api_url)
+    content = json.loads(req.read())
+    branches = [repo_branch["name"] for repo_branch in content]
+    if version_only:
+        branches = [
+            version_branch
+            for version_branch in branches
+            if version_branch.startswith("grass")
+        ]
+    branches.sort()
```

Comment:
This simple sort wouldn't not work beyond v9 for the purpose of picking the latest available branch. "natural sort" is needed. However, I'm suggesting to remove this anyway. (Useful in general, so perhaps keep the ocde it around.)